utils/calico-uts.Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:14.04
+
+MAINTAINER Alex Chan <alexchan@projectcalico.org>
+
+RUN apt-get update
+RUN echo "APT::GET::Assume-Yes \"true\";" >> /etc/apt/apt.conf
+
+# tox installs python-posix-spawn and python-etcd from Git
+RUN apt-get install git
```

Comment:
All these separate RUN stanza will each lay down a filesystem layer.  It's usually better to put them all together to minimize the size of the image.